view/frontend/web/js/view/payment/method-renderer/omise-cc-googlepay-method.js

```diff
@@ -349,12 +349,12 @@ define(
              * Send charge creation request and finalize payment.
              */
             processPayment: function() {
-                var failHandler = klass.buildFailHandler(klass);
+                let failHandler = klass.buildFailHandler(klass);
                 klass.getPlaceOrderDeferredObject()
                     .fail(failHandler)
                     .done(function(orderId) {
-                        var serviceUrl = klass.getMagentoReturnUrl(orderId);
-                        var storageFailHandler = klass.buildFailHandler(klass);
+                        let serviceUrl = klass.getMagentoReturnUrl(orderId);
+                        let storageFailHandler = klass.buildFailHandler(klass);
```

Comment:
Yeah. I know. I am using `let` just to be safe so that I don't use `const` for variable that will change in other parts. I have changed some from `let` to `const`.